verification.cs

```diff
@@ -0,0 +1,13 @@
+// .NET 5
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+var key = "";
+var userId = "";
+
+var now = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+using var hmac = new HMACSHA256(Encoding.UTF8.GetBytes(key));
+var hashBytes = hmac.ComputeHash(Encoding.UTF8.GetBytes($"{userId}-{now}"));
+var hashString = BitConverter.ToString(hashBytes).Replace("-", "").ToLower();
```

Comment:
just checking: have you checked that this code "works" via the verification checker? my concern is that removing the dash here will fail verification

Reply:
Yes :)

![image](https://user-images.githubusercontent.com/16026608/106043358-e3a35500-60a3-11eb-90eb-9546ec64f776.png)

Reply:
This is how it would look like without removing the dashes

```
e2-d8-4f-37-8e-34-e7-64-08-37-66-0c-3c-ea-cc-7a-c2-72-34-95-3d-ab-cb-2d-41-73-4e-bc-11-d5-67-ec-1611776076
```

Reply:
ok great-- this makes sense and holy moly dashes!